### PR TITLE
WEBSITE-335 Tags followup

### PR DESCRIPTION
**Pull request by DavideD**
https://hibernate.atlassian.net/browse/WEBSITE-335

* Flex => Bean Validation
* Filters => Hibernate ORM
* Increase space between tags

@emmanuelbernard I've tried to apply the label css to the tags but it does not look good. Mainly because it becomes hard to read the tags (It is actually already quite hard to read it under the title)

I can create a branch where I use the label style for the tags if you want to have a look at it

### CODE REVIEW

**Comment by emmanuelbernard** [comment]
We should go ahead with the plural form while we are at it.

**Comment by emmanuelbernard** [comment]
The CSS can be moved to blog-styles now.

**Comment by emmanuelbernard** [comment]
The tag space small improvement that can go ahead. We can refine these things further later anyways. I'll let you retrofit the PR with the plural and the css thing.

**Comment by DavideD** [comment]
Rebased over 
https://github.com/hibernate/in.relation.to/pull/26

Move the blog CSS in the new related file and fix the tags changing them in the plural form

File: redirects/.htaccess_posts [hunk 1]
```diff
@@ -3804,47 +3804,47 @@ RewriteRule ^tag/jboss5$ /jboss-as/ [R=301,L,E=nocache:1]
...
+RewriteRule ^tag/Persistence$ /discussions/ [R=301,L,E=nocache:1]
 RewriteRule ^tag/Photography$ /off-topic/ [R=301,L,E=nocache:1]
 RewriteRule ^tag/PicketLink$ /picketlink/ [R=301,L,E=nocache:1]
 RewriteRule ^tag/Portable\+Extensions$ /cdi/ [R=301,L,E=nocache:1]
 RewriteRule ^tag/PostgreSQL$ /hibernate-orm/ [R=301,L,E=nocache:1]
-RewriteRule ^tag/Proposal$ /discussion/ [R=301,L,E=nocache:1]
-RewriteRule ^tag/release$ /release/ [R=301,L,E=nocache:1]
-RewriteRule ^tag/Release$ /release/ [R=301,L,E=nocache:1]
-RewriteRule ^tag/ReleaseNotes$ /release/ [R=301,L,E=nocache:1]
+RewriteRule ^tag/Proposal$ /discussions/ [R=301,L,E=nocache:1]
```
Comment (gunnarmorling) [inline redirects/.htaccess_posts]: I think "discussion" is better than the plural.

**Pull request merged**